### ISSUE TITLE
Issue on page /Part_3_Python/M3/tasks/ind_1/genotype/instructions.html

### ISSUE THREAD

**Issue by eylulustaoglu** [Purdue-ENGR-13300/2025-fall#16]
The flowchart image in section 9.3.3. does not have alt text.

**Comment by TylerSivertsen** [comment]
Fixed in https://github.com/Purdue-ENGR-13300/content/commit/54eba07fabc07d2784a5c1c3d3cbfd94a67987de
